src/hooks/util.ts

```diff
@@ -67,3 +70,29 @@ export const getResourceByProductId = (productId: string): { type: ResourceType;
     .toLowerCase() as ResourceType
   return { type: resourceType, target: productTarget }
 }
+
+export const useToastMessage = () => {
+  const { formatMessage } = useIntl()
+  const toast = useToast()
+  const toastMessage = (options: {
```

Comment:
import { UseToastOptions } from  '@chakra-ui/react'
there is some types can extends